.github/workflows/Lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Build CSS
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "20"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Prettier Check
```

Comment:
Would we be able to incorporate this into the action that's already set up to build the CSS when we push changes? That way we could keep them both in the same action

Reply:
Yeah you can have multiple jobs per workflow file. I just assumed you wanted them separate because of how you named the file. Should I change this PR to add this job to the existing workflow?

Reply:
Yeah, lets add this job to the current workflow file, and you can rename that to what seems best if you'd like!

Reply:
Ok cool I merged them into one file. because we want prettier to always run I removed the `paths: 'scss/**'` and used the `stefanzweifel/git-auto-commit-action` which automatically only creates a commit if the output css actually changed. I noticed that you had it configured to only run on the main branch, I'm not sure if that was intentional, it seams like you would always want that to run but if you did only want it to run only on main then I can add that back in for that job. I also noticed we had an outdated version of `actions/checkout` and `actions/setup-node` so I updated them.

Reply:
Lets merge and see how it goes! Thanks for the help with this!